workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/EntityGameObjectLinkerTests.cs

```diff
@@ -49,8 +49,7 @@ public void TearDown()
         [Test]
         public void LinkGameObjectToEntity_adds_SpatialOSComponent_component()
         {
-            entityGameObjectLinker.LinkGameObjectToEntity(testGameObject, testEntity, testSpatialEntityId,
-                viewCommandBuffer);
+            entityGameObjectLinker.LinkGameObjectToEntity(testGameObject, testEntity, viewCommandBuffer);
```

Comment:
I'd expect this test to fail unless there's something that adds the `SpatialEntityId` to `testEntity`?

Reply:
Hmm, I did run the tests, maybe the code wasn't reloaded? I agree with you, taking a look